### ISSUE TITLE
Make sure data-size="..." is used to fetch the optimized image, or use the original

### ISSUE THREAD

**Issue by alberto56** [dcycle/thumbor-example#17]
get the latest master commit of https://github.com/dcycle/thumbor-example

./scripts/deploy.sh

Look at the source code of ./website-with-large-image/optimized.html

Add the following somewhere on the page:

    <img data-src="/large-image.jpg" width="300" data-size="200x" alt="Columns in courtyard">
    <img data-src="/large-image.jpg" width="300" data-size="this-size-is-not-in-mapping" alt="Columns in courtyard">

Go to http://0.0.0.0:8706/optimized.html

You will now see four images:

* http://0.0.0.0:8706/this-unversioned-large-image-is-purposefully-not-mapped.jpg
* Three times http://0.0.0.0:8705/UvEI35-AIBlY3dylFvnpwS6czDI=/500x0/smart/webserver/large-image.jpg

Your task
-----  

Instead, we want 

* http://0.0.0.0:8706/this-unversioned-large-image-is-purposefully-not-mapped.jpg
* http://0.0.0.0:8705/UvEI35-AIBlY3dylFvnpwS6czDI=/500x0/smart/webserver/large-image.jpg
* http://0.0.0.0:8705/large-image.jpg
* http://0.0.0.0:8705/yKo6KvMfAcHxB87ACITgjsRjRQY=/200x0/smart/webserver/large-image.jpg

In other words, data-size="..." seems to be ignored.

* If we can map, say 200x or 500x, then use the mapped size.
* If we cannot map, say data-size="this-size-is-not-in-mapping, then use the original. We seem to always be using 500x.

**Comment by alberto56** [comment]
Done, closing
